bionty/base/__init__.py

```diff
@@ -1,10 +1,36 @@
-"""Bionty base.
+"""Bionty base is a standalone version of Bionty without lamindb support.
 
-Import the package::
+Overview
+========
 
-   import bionty.base as bt
+Bionty base is designed for scripts that need to access public ontologies without more advanced lamindb support.
+If any public ontology needs to be extended, use [LaminDB](https://lamin.ai/docs) together with Bionty.
```

Comment:
I'd remove this sentence. People will know this if they ever end up in this nested piece of the docs.